appserver/distributions/payara-ml/pom.xml

```diff
@@ -108,6 +116,30 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>DeploySnapshot</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>DeploySnapshot</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.7</version>
```

Comment:
Should be 1.6.8 as sonatype always makes a big point of projects using the latest version